src/main/java/firebase/sdk/dashboard/api/ReleaseResource.java

```diff
@@ -28,57 +32,25 @@
 @Consumes(MediaType.APPLICATION_JSON)
 public class ReleaseResource {
 
-  private String platform;
-  // TODO: Delete once actual implementation is done
-  private ArrayList<String> sdks = new ArrayList<>();
+  private Platform platform;
+  private final ReleaseDao RELEASEDAO = new PlatformReleaseDaoDatastore();
+  private final SDKDao SDKDAO = new SDKDaoDatastore();
 
   public ReleaseResource(String platform) {
-    this.platform = platform;
-    // Dummy Code for API
-    sdks.add("firebase-common");
-    sdks.add("firebase-common-ktx");
-    sdks.add("firebase-ml");
-    sdks.add("firebase-database");
-    sdks.add("firebase-auth");
-    sdks.add("firebase-components");
-    sdks.add("firebase-iid");
-    sdks.add("firebase-analytics");
-    sdks.add("measurement");
-    sdks.add("measurement_api");
-    sdks.add("tagmanager");
-    sdks.add("tagmanager-api");
-    sdks.add("firebase-ads");
-    sdks.add("phenotype");
-    sdks.add("functions");
-    sdks.add("transport-runtime");
-    sdks.add("crashlytics-gradle");
-    sdks.add("firebase-components"); 
+    this.platform = Platform.get(platform);
```

Comment:
so when this pr is merged into master, the endpoints will not have any data until data is migrated to the datastore?